db/primary_migrate/20230627213457_add_notification_phone_configurations_table.rb

```diff
@@ -0,0 +1,10 @@
+class AddNotificationPhoneConfigurationsTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :notification_phone_configurations do |t|
+      t.references :in_person_enrollment, null: false, index: { name: 'index_notification_phone_configurations_on_enrollment_id' }
```

Comment:
can we update this to have a unique index on the enrollmend ID to help enforce the "zero or one"?

Reply:
Done in 968d19ba202d6c68abefc3fa0c7b7e7359e4ff97